packages/mobile/src/harmony-native/components/input/TextInput/TextInput.tsx

```diff
@@ -334,7 +335,10 @@ export const TextInput = forwardRef(
                       underlineColorAndroid='transparent'
                       aria-label={ariaLabel ?? labelText}
                       style={css({
-                        flex: 1,
+                        padding: 0,
+                        // Need absolute height to ensure consistency across platforms
```

Comment:
whatt..
1. why do we need `paddingVertical` after already specifying `padding: 0`?
2. we don't need special cases for the other sizes?

Reply:
oh yes i accidently deleted the comment. android has default padding on text-input